src/cockpit/buffered.pyfragment

```diff
@@ -1,6 +1,6 @@
 
 class CockpitBufferedProtocol(CockpitBaseProtocol, asyncio.BufferedProtocol):
-    '''A hopefully-reasonably-efficient implementation of the Cockpit frame protocol
+    """A hopefully-reasonably-efficient implementation of the Cockpit frame protocol
```

Comment:
Seems black also prefers """, interesting! Although this is still inconsistent in our codebase I bet, so linting it some day would be nice